net.retrodeck.retrodeck.yml

```diff
@@ -0,0 +1,1146 @@
+      - --disable-werror
+      - --target-list=i386-softmmu
+    make-args:
+      - qemu-system-i386
+    post-install:
+      - |-
+        for px in 16 32 48 64 128 256 512; do
+          install -Dm644 ../ui/icons/xemu_${px}x${px}.png /app/share/icons/hicolor/${px}x${px}/apps/app.xemu.xemu.png
+        done
+      - install -Dm644 ../ui/icons/xemu.svg /app/share/icons/hicolor/scalable/apps/app.xemu.xemu.svg
+      - desktop-file-install ../ui/xemu.desktop --dir /app/share/applications
+      - sed -i 's/^Icon=xemu/Icon=app.xemu.xemu/' /app/share/applications/xemu.desktop
+      - mv qemu-system-i386 /app/bin/xemu
+      - mkdir -p /app/share/licenses/xemu
+      - cd .. && python3 scripts/gen-license.py > /app/share/licenses/xemu/LICENSE.txt
+    sources:
+      - type: git
+        url: https://github.com/mborgerson/xemu.git
+        tag: gh-release/0.6.2-90-g6f507c80af
+        commit: 6f507c80af546e0f5d316622d20f78820423ce6e
+        x-checker-data:
+          type: json
+          url: https://api.github.com/repos/mborgerson/xemu/releases/latest
+          tag-query: .tag_name
+          version-query: $tag | sub("^gh-release/"; "")
+          timestamp-query: .published_at
+          is-main-source: true
+
+  # XEMU - END
+
+  # MELONDS - START
+  # https://github.com/flathub/net.kuribo64.melonDS
+  # ちっちゃい、かわいい！
+
+  - name: libslirp
+    buildsystem: meson
+    sources:
+      - type: git
+        url: https://gitlab.freedesktop.org/slirp/libslirp.git
+        tag: v4.4.0
+  - name: melonds
+    buildsystem: cmake-ninja
+    builddir: true
+    build-options:
+      arch:
+        aarch64:
+          config-opts:
+            - -DENABLE_OGLRENDERER=OFF
+    sources:
+      - type: git
+        url: https://github.com/Arisotura/melonDS.git
+        commit: cce9de5ee1f51e906a644922b2ada6644d81287a
+
+  # MELONDS - END
+
+  # CITRA - START
+  # https://github.com/flathub/org.citra_emu.citra
+
+  - name: citra
+    buildsystem: cmake-ninja
+    builddir: true
+    config-opts:
+      - '-DCMAKE_BUILD_TYPE=Release'
+      - '-DENABLE_QT_TRANSLATION=ON'
+      - '-DCITRA_ENABLE_COMPATIBILITY_REPORTING=ON'
+      - '-DENABLE_COMPATIBILITY_LIST_DOWNLOAD=ON'
+      #- '-DENABLE_FFMPEG_VIDEO_DUMPER=ON'
+      # Disabled because of this: https://github.com/citra-emu/citra/issues/5949
+      - '-DENABLE_FDK=ON'
+    cleanup:
+      - /share/man
+      - /share/pixmaps
+    post-install:
+      - >-
+        install -Dm644 ../dist/citra.svg
+        /app/share/icons/hicolor/scalable/apps/citra.svg
+      - >-
+        install -Dm644 ../dist/icon.png
+        /app/share/icons/hicolor/512x512/apps/citra.png
+    sources:
+      - type: git
+        url: https://github.com/citra-emu/citra-nightly.git
+        tag: nightly-1762
+        commit: 1382035d4de5ada310c7f4fd00f1836b16ab74fb
+        disable-shallow-clone: true
+        x-checker-data:
+          type: git
+          tag-pattern: ^(nightly-\d+)$
+          is-main-source: true
```

Comment:
There should be only one `is-main-source` in the entire manifest.

Reply:
Removed from upstream